spec/support/tests/database_adapters/config/postgresql.yml

```diff
@@ -0,0 +1,19 @@
+default: &default
+  adapter: postgresql
+  encoding: unicode
+  pool: <%= ENV.fetch("RAILS_MAX_THREADS", 5) %>
+  host: <%= ENV.fetch("DB_HOST", "localhost") %>
+  username: <%= ENV.fetch("DB_USER", "postgres") %>
```

Comment:
You might want to double-check that this doesn't break local development. I don't know if we can assume a `postgres` user always exists locally (for instance, I think on Mac, when using Postgres from Homebrew, a role is created that matches your username). We might need to use `ENV["CI"]` to determine whether to use `postgres` or not.

Reply:
> I don't know if we can assume a postgres user always exists locally (for instance, I think on Mac, when using Postgres from Homebrew, a role is created that matches your username).

For this kind of scenario, the user can configure the necessary variable. Example:

```
DATABASE_ADAPTER=postgresql DB_USER=vsppedro bundle exec appraisal rails_6_0 rake spec:acceptance
```
It will be used vsppedro as username instead of postgres, because the method .fetch tries to catch the environment variable first, if returns nill it uses the second value.

A double-check would be nice. I'll ask some friends to run this command locally. I only can test this on Linux. 😅

Reply:
Yeah, I guess it's not very common to run the Postgres tests locally, so if the developer has to supply `DB_USER` it's probably a minor convenience. So I'd be fine with this.